issuer_test.go

```diff
@@ -0,0 +1,102 @@
+package issuer
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+	"gopkg.in/square/go-jose.v2"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestIssueCard(t *testing.T) {
+	file, err := os.Open("verifiable_credential.json")
```

Comment:
For this test, can we use a JSON string or byte slice as a variable, so the test doesn't depend on a file?